docs/features/kubernetes/proxy.md

```diff
@@ -60,6 +67,8 @@ the value in the header will be targeted.
 Then the request will be forwarded verbatim (but with the endpoint's base URL
 prefix stripped) to the cluster.
 
+The proxy will also interpret the `Backstage-Kubernetes-Authorization` header as the `Authorization` header to use when forwarding a request to a target cluster.
```

Comment:
Can we say something like this in the 'Authentication' section below? Right now it says "Bearer tokens will be forwarded as-is" and that feels _slightly_ misleading -- sure, the token itself won't be modified, but the header that contains it will be modified before making the proxied request.

Reply:
Added to the Authentications section with my last section update. Still need to address how i access backstage id tokens in the code snippet. l will get to that tomorrow. Added.